dcc-portal-ui/config/webpack.config.dev.js

```diff
@@ -5,7 +5,7 @@ var CaseSensitivePathsPlugin = require('case-sensitive-paths-webpack-plugin');
 var paths = require('./paths');
 
 module.exports = {
-  devtool: 'cheap-module-eval-source-map',
+  devtool: 'source-map',
```

Comment:
sounds good, as long as the default value is still `'cheap-module-eval-source-map'`